Source/Core/DolphinWX/MainMenuBar.cpp

```diff
@@ -140,6 +140,7 @@ wxMenu* MainMenuBar::CreateMovieMenu() const
 
   movie_menu->Append(IDM_RECORD, _("Start Re&cording Input"));
   movie_menu->Append(IDM_PLAY_RECORD, _("P&lay Input Recording..."));
+  movie_menu->Append(IDM_STOP_RECORD, _("Stop Play/Recording Input"));
```

Comment:
Is it possible to make this change name at runtime? The current name feels a bit confusing to me. Also, please replace "Play" with "Playing" - that would probably also make it less confusing.